### ISSUE TITLE
Add 'Key' prefix to span tag names

### ISSUE THREAD

**Issue by knusbaum** [DataDog/dd-trace-go#579]
In tracer version 2, we want to prefix span tag key constant (SpanType, etc) with "Key*" (KeySpanType, etc) to more easily differentiate between constants representing tag values and constants representing keys.
https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/ext/tags.go

**Comment by gbbr** [comment]
I agree. We could use that pattern. We could also do things like `KeyPriorityKeep` and `KeyPriorityDrop` instead of `Manual{Keep,Drop}`.

The other thing is that this whole `ext` package doesn't really make sense. It might make more sense to move all of these constants into the `tracer` package, especially if we'll be prefixing them.

**Comment by darccio** [comment]
This is not going to be implemented unless we come with a proposal with more reasons for doing it.
